src/actions/operator.ts

```diff
@@ -771,7 +771,7 @@ class ToggleCaseVisualBlockOperator extends BaseOperator {
 
 @RegisterAction
 class ToggleCaseWithMotion extends ToggleCaseOperator {
-  public keys = ['g', '~'];
+  public keys = [['g', '~']];
```

Comment:
This change shouldn't be necessary

Reply:
Thanks for the review.
If I don't make this change, I get `ts2416` error.
I suppose this is because `ToggleCaseWithMotion` inherits `ToggleCaseOperator` and`ToggleCaseOperator` implements `keys` of type `string[][]`.
Do you know how to solve this?

Reply:
Oh I see - nope, that's fine